CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/providers/EsriMapProvider.java

```diff
@@ -44,9 +47,23 @@ public PImage getTile(final Coordinate coordinate) {
 
     @Override
     public String[] getTileUrls(final Coordinate coordinate) {
-        final String url = String.format(
-                "%s/export?f=image&format=png&size=256,256&imagesr=3857&bbox=%s",
-                getMapServer(), getBoundingBox(coordinate));
+        final String url;
+        switch (getMapServerType()) {
+            case TILE:
+                url = String.format(
+                        "%s/tile/%s",
+                        getMapServer(), getZoomString(coordinate));
+                break;
+            case EXPORT:
+                url = String.format(
+                        "%s/export?f=image&format=png&size=256,256&imagesr=3857&bbox=%s",
+                        getMapServer(), getBoundingBox(coordinate));
+                break;
+            default:
+                url = null;
```

Comment:
Any place we introduce nulls means that we need to check for it downstream. I'm not sure what other options we can use; more of an observation comment.

Reply:
Passing null's to Unfolding Maps doesn't break anything, but it does throw null pointer exception for each tile request. I updated this to return an empty string, which instead warns you that the resource cannot be found.